static/scss/_button.scss

```diff
@@ -23,11 +23,12 @@
   padding: 10px 15px;
   min-width: 120px;
   text-transform: uppercase;
-  box-shadow: 2px 2px 8px grey;
-  font-weight: 600;
+  font-weight: 400;
+  font-size: 13px;
```

Comment:
For some reason I had though `pt` was preferable to `px`, but it looks like this is correct: http://stackoverflow.com/questions/3557260/should-i-use-pt-or-px

We should switch over the other font sizes which use `pt` whenever we get around to it